doc/getting_started/installation_mac.rst

```diff
@@ -9,6 +9,7 @@ After completing these steps, you will be able to compile and run your Zephyr
 applications on the following Mac OS version:
 
 * Mac OS X 10.11 (El Capitan)
+* macOS Sierra 10.12
 
 Developing for Zephyr on OS X generally requires you to build the
```

Comment:
Yes, the transition from "OS X" to "macOS" (it was only called Mac OS X years and years ago) happened in 10.12